patches/react-native-svg+13.9.0.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/react-native-svg/src/css.tsx b/node_modules/react-native-svg/src/css.tsx
+index f567b80..eb75078 100644
+--- a/node_modules/react-native-svg/src/css.tsx
++++ b/node_modules/react-native-svg/src/css.tsx
+@@ -695,14 +695,22 @@ export function SvgCssUri(props: UriProps) {
+   const { uri, onError = err, onLoad } = props;
+   const [xml, setXml] = useState<string | null>(null);
+   useEffect(() => {
+-    uri
+-      ? fetchText(uri)
++    const fetchController = new AbortController();
```

Comment:
Same with this - as svg is a pretty common file across the whole project, it just feels a bit risky making too many changes

Reply:
I noticed `SvgCssUri` in react-native-svg always leaks memory. So it was important to fix this.